### PR TITLE
test-battery: pycodestyle

### PR DESCRIPTION
**Pull request by oliver-sanders**
The python `pep8` package has been renamed `pycodestyle` to avoid confusion.

This fixes `t/syntax/00-lib.t` which will fail with newer versions of `pep8` due to a depreciation notice.

### CODE REVIEW

**Comment by matthewrmshin** [comment]
(Would this work on our site?)

**Comment by oliver-sanders** [comment]
No, hence I have left the `pep8` test in for compatibility.

File: lib/python/rosie/ws.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def index(self, *_):
         """Provide the index page."""
         try:
             return self._render()
-        except:
+        except (KeyError, AttributeError, jinja2.exceptions.TemplateError):
```
Comment (oliver-sanders) [inline lib/python/rosie/ws.py]: `jinja2.exceptions.TemplateError` might be sufficient here?

File: lib/python/rose/run.py [hunk 2]
```diff
@@ -202,8 +202,8 @@ def run(self, opts, args):
         finally:
             # Close handle on specific log file
             try:
-                self.event_handler.contexts.get(uuid).handle.close()
-            except:
+                self.event_handler.contexts[uuid].handle.close()
+            except (KeyError, IOError, AttributeError):
```
Comment (oliver-sanders) [inline lib/python/rose/run.py]: possibly `TypeError` as well?

File: lib/python/rosie/ws.py (hunk 1 above)
Comment (matthewrmshin) [reply lib/python/rosie/ws.py]: Most likely. The client will get a 500 in the worst case.

File: lib/python/rose/run.py (hunk 2 above)
Comment (matthewrmshin) [reply lib/python/rose/run.py]: I can't see how.

**Comment by matthewrmshin** [comment]
@stevewardle do you mind doing a sanity check for us?

**Comment by stevewardle** [comment]
Looks perfectly reasonable to me

**Pull request merged**